### ISSUE TITLE
99.9% percent of sample predicted label are -1

### ISSUE THREAD

**Issue by zxsimple** [Angel-ML/angel#734]
I trained a DeepFM model with AUC `0.84`, while predicting on the training data, 99.9% and more samples predicted as -1.

![20190419-090251(eSpace)](https://user-images.githubusercontent.com/10580990/56399646-fdcec300-6281-11e9-9d6c-3cd39b63f5e8.png)

**Comment by wangcaihua** [comment]
can you post you training hyper-parameter?

the reason could be that the regularization parameters is so large that the model is under trained

I gauss that the true positive/negative rate are also abnormal.

**Comment by zxsimple** [comment]
The **FM** model has this problem, after I predicting on **WAD** model, the prediction data seems much reasonable. The Pos/Neg ratio is 1:3 on both models.

Here is my **FM** model parameters:
```
... ...
--ml.model.size 9257 \
--ml.feature.index.range 9257 \
--ml.data.type libsvm \
--ml.learn.rate 0.1 \
--ml.reg.l2 0.2 \
--ml.fm.field.num 61 \
--ml.fm.rank 8 |
--ml.inputlayer.optimizer ftrl \
--ml.epoch.num 10 \
--ml.data.label.trans.class PosNegTrans \
-- ml.data.label.trans.threshold 0.5 \
... ...
```
